recipes/mariadb-connector-c/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build && cd build
+
+:: Set INSTALL_DOCREADMEDIR to a junk path to avoid installing the README into PREFIX
+cmake %CMAKE_ARGS% ^
+      -GNinja ^
+      -DCMAKE_BUILD_TYPE="Release" ^
+      -DCMAKE_C_FLAGS="-I%LIBRARY_INC%" ^
+      -DCMAKE_CXX_FLAGS="-I%LIBRARY_INC%" ^
+      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -DINSTALL_DOCREADMEDIR_STANDALONE="%cd%/junk" ^
+      -DINSTALL_DOCDIR="%cd%/junk" ^
+      ..
+if errorlevel 1 exit 1
+
+ninja
+if errorlevel 1 exit 1
+
```

Comment:
Is there a make check or ctest that we can run here?

Reply:
Contained within the build there is a directory called unittest with two sub directories libmariadb and mytap.

I think these would be the tests that you're referring to. I am unsure of how to actually add that to the build script.

Reply:
Take a look at https://github.com/mariadb-corporation/mariadb-connector-c/blob/3.3/travis.sh for some inspiration.

Reply:
I added ctest to build.sh and build.bat